api/migrations/schema/Version20220501060227.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Diese migration is fehlgeschlagen bei meinem lokalen stand:
```
 In ExceptionConverter.php line 54:
ecamp3-api     |                                                                                
ecamp3-api     |   An exception occurred while executing a query: SQLSTATE[23503]: Foreign key  
ecamp3-api     |    violation: 7 ERROR:  insert or update on table "abstract_content_node_owne  
ecamp3-api     |   r" violates foreign key constraint "fk_8e710ab4f886581c"                     
ecamp3-api     |   DETAIL:  Key (rootcontentnodeid)=(3e67389a5898) is not present in table "co  
ecamp3-api     |   ntent_node_columnlayout".                                                    
ecamp3-api     |                                                                                
ecamp3-api     | 
ecamp3-api     | In Exception.php line 30:
ecamp3-api     |                                                                                
ecamp3-api     |   SQLSTATE[23503]: Foreign key violation: 7 ERROR:  insert or update on table  
ecamp3-api     |    "abstract_content_node_owner" violates foreign key constraint "fk_8e710ab4  
ecamp3-api     |   f886581c"                                                                    
ecamp3-api     |   DETAIL:  Key (rootcontentnodeid)=(3e67389a5898) is not present in table "co  
ecamp3-api     |   ntent_node_columnlayout".                                                    
ecamp3-api     |                                                                                
ecamp3-api     | 
ecamp3-api     | In Connection.php line 72:
ecamp3-api     |                                                                                
ecamp3-api     |   SQLSTATE[23503]: Foreign key violation: 7 ERROR:  insert or update on table  
ecamp3-api     |    "abstract_content_node_owner" violates foreign key constraint "fk_8e710ab4  
ecamp3-api     |   f886581c"                                                                    
ecamp3-api     |   DETAIL:  Key (rootcontentnodeid)=(3e67389a5898) is not present in table "co  
ecamp3-api     |   ntent_node_columnlayout".                                                    
ecamp3-api     |                                                                
```
[Version202205150310_data.txt](https://github.com/ecamp/ecamp3/files/8695005/Version202205150310_data.txt)

Ist die Frage ob man das noch anschauen will.
Sobald alle ihre db's gespühlt haben, ist das wieder ok.